_posts/2021-10-05-str-methods.md

```diff
@@ -106,25 +173,25 @@ Welcome to Share to Learn
 Welcome to Learn to Share
 ```
 
+<div id="format_map"/>
+
 **format_map()**	Formats specified values in a string using a dictionary
 ```python
->>> lunch = {"Food":
-"Pizza", "Drink": "Wine"}
->>> print("Lunch: {Food},
-{Drink}".format_map(lunch))
-Lunch: Pizza, Wine
+>>> lunch = {
+            "Food": "Pizza", 
+            "Drink": "Juice"}
+>>> print("Lunch: {Food}, {Drink}".format_map(lunch))
+Lunch: Pizza, Juice
```

Comment:
Could you please add `print("Lunch: {Food}, {Drink},{dessert}".format_map(lunch))` too as an example and it's output, which would be `KeyError: 'dessert'` . This would help the reader to understand and appreciate what is being done in `class Default(dict)`

Reply:
Done